src/test/spec/app/private/scripts/controllers/announcement/createAnnouncementController.js

```diff
@@ -48,193 +45,236 @@ describe('Controller: createAnnouncementCtrl', function () {
     createAnn = $controller('createAnnouncementCtrl', {
       $scope: scope,
       $rootScope: $rootScope,
-      dashboardService: dashboardService,
       toasterService: toasterService,
       permissionsService: permissionsService,
       $state: $state,
       $modal: modal
     })
   }))
 
-  it('should initialize create announcement Modal', function (done) {
+  it('should initialize create announcement Modal', function () {
     spyOn(createAnn, 'initializeModal').and.callThrough()
     createAnn.initializeModal()
     timeout.flush(100)
-    done()
+    expect(createAnn.initializeModal).toBeDefined()
```

Comment:
should this also check if the button is enabled and if the recipient has been selected? @nilesh-more @loganathan1989 

https://github.com/project-sunbird/sunbird-portal/pull/100/files#diff-23f1b15be05b1d6e2705f1f2a2bed2a8R53